wallet/src/main/scala/org/bitcoins/wallet/internal/FundTransactionHandling.scala

```diff
@@ -138,20 +129,13 @@ trait FundTransactionHandling extends WalletLogger { self: Wallet =>
           logger.info(s"UTXO $index details: ${utxo.output}")
       }
 
-      val inputs =
```

Comment:
So basically we are moving from doing this explicitly to use the built in code inside of `ShufflingNonInteractiveFinalizer`?

Reply:
Yes, these exact lines are already in there